crates/astria-sequencer/src/fees/state_ext.rs

```diff
@@ -532,6 +541,20 @@ pub(crate) trait StateWriteExt: StateWrite {
 
 impl<T: StateWrite> StateWriteExt for T {}
 
+/// Creates `abci::Event` of kind `tx.fees` for sequencer fee reporting
+fn construct_tx_fee_event(fee: &Fee) -> Event {
+    Event::new(
+        "tx.fees",
+        [
+            ("actionName", fee.action_name.to_string()).index(),
+            ("asset", fee.asset.to_string()).index(),
+            ("feeAmount", fee.amount.to_string()).index(),
+            ("sourceTransactionId", fee.source_transaction_id.to_string()).index(),
```

Comment:
This shouldn't be needed when correctly attached to a given tx?

Reply:
removed in a72a7c9